### PR TITLE
create event migration

### PR DESCRIPTION
**Pull request by Rexben001**
Create migrations and association
https://dbdiagram.io/d/5d3c5d91ced98361d6dd0e45

### CODE REVIEW

File: database/migrations/20200102171324-create-event.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      desc: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      creator: {
+        type: Sequelize.UUIDV4,
+        allowNull: true
```
Comment (Oyekunle-Mark) [inline database/migrations/20200102171324-create-event.js]: Creator is a foreign key that references user id, I am not sure it should allow null

File: database/migrations/20200102171324-create-event.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      },
+      start_date: {
+        type: Sequelize.DATE,
+        allowNull: true
+      },
+      end_date: {
+        type: Sequelize.DATE,
+        allowNull: true
+      },
```
Comment (Oyekunle-Mark) [inline database/migrations/20200102171324-create-event.js:43]: Start date and end date should not be null

File: database/migrations/20200102172244-create-ticket.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4
+      },
+      ticket_no: {
+        type: Sequelize.INTEGER,
+        allowNull: false
+      },
```
Comment (Oyekunle-Mark) [inline database/migrations/20200102172244-create-ticket.js:14]: Ticket no here should be uuid. We need a unique hash looking string. Maybe just a string that has the event name and user details somewhere.

File: database/migrations/20200102171324-create-event.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+      title: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      desc: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
```
Comment (Oyekunle-Mark) [inline database/migrations/20200102171324-create-event.js:26]: The default string length is 255, is there a type that supports something longer for event description?

File: database/migrations/20200102172244-create-ticket.js (hunk 3 above)
Comment (Rexben001) [reply database/migrations/20200102172244-create-ticket.js:14]: I was following the db diagram, I'd change it to String

**Pull request merged**